Parserversion_1/src/Program.cs

```diff
@@ -15,32 +15,32 @@ static void Main(string[] args)
             Program obj = new Program();
             Console.WriteLine("Main Role für 10 Helden");
             obj.loadfile();
-            
+
         }
-       public void loadfile()
+        public void loadfile()
         {
 
             List<string> list = new List<string>();
-            using (StreamReader Reader = new StreamReader("..\\..\\..\\..\\test.csv"))
+            using (StreamReader Reader = new StreamReader(@".\test.csv"))
```

Comment:
The `@` tell the c# compiler the string is a string with special characters. So you don't need to escape the `\`. See more [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim)